converter/internal/prometheusconvert/relabel.go

```diff
@@ -49,7 +49,7 @@ func appendDiscoveryRelabel(pb *prometheusBlocks, relabelConfigs []*prom_relabel
 	pb.discoveryRelabelBlocks = append(pb.discoveryRelabelBlocks, newPrometheusBlock(block, name, label, "", ""))
 
 	return &disc_relabel.Exports{
-		Output: newDiscoveryTargets(fmt.Sprintf("discovery.relabel.%s.targets", label)),
+		Output: newDiscoveryTargets(fmt.Sprintf("discovery.relabel.%s.output", label)),
```

Comment:
Found this issue thanks to adding an attempt to load flow config in `testing.go`.